Imagekit.UnitTests/ImageKitTestCasesNonAsync.cs

```diff
@@ -724,6 +726,31 @@ public void SimilarImage_getHammingDistance_expectedSuccessWith()
             int hammingDistance = imagekit.PHashDistance("33699c96619cc69e", "968e978414fe04ea");
             Assert.Equal(2, hammingDistance);
         }
+        [Fact]
+        public void AuthenticationParamCheck()
+        {
+            var imagekit = new ImagekitClient(GOOD_PUBLICKEY, GOOD_PRIVATEKEY, GOOD_URLENDPOINT);
+            string token = "your_token";
+            string expire = "1582269249";
+            AuthParamResponse authParams = imagekit.GetAuthenticationParameters(token, expire);
+            Assert.Equal(token, authParams.token);
+            Assert.Equal(expire, authParams.expire);
+            Assert.Equal("9ce92ac594783df383d87a25ffad2b8145936d5c", authParams.signature);
```

Comment:
Please use same values as in this Node.js test case - https://github.com/imagekit-developer/imagekit-nodejs/blob/master/tests/unit.js#L16
`9ce92ac594783df383d87a25ffad2b8145936d5c` signature for `token=your_token` and `expire= 1582269249` is incorrect. The `signature` should have been `e71bcd6031016b060d349d212e23e85c791decdd` using `private_key_test` private key.